infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/VcsSshKeysProvisioner.java

```diff
@@ -58,32 +58,30 @@
  *   <li>create K8s Secrets with <a
  *       href=https://github.com/kubernetes/kubernetes/blob/7693a1d5fe2a35b6e2e205f03ae9b3eddcdabc6b/pkg/apis/core/types.go#L4458">SSH
  *       Key type</a> for each of SSH keys;
- *   <li>secrets are mounted on each container by path '/etc/ssh/{sshKeyName}/ssh-privatekey' as a
- *       file;
+ *   <li>secrets are mounted on each container by path
+ *       '/etc/ssh/private/{sshKeyName}/ssh-privatekey' as a file;
  *   <li>ConfigMap with SSH settings is created and mounted to container by path
  *       '/etc/ssh/ssh_config'.
  * </ul>
  *
  * @author Vitalii Parfonov
+ * @author Vlad Zhukovskyi
  */
 public class VcsSshKeysProvisioner implements ConfigurationProvisioner<KubernetesEnvironment> {
```

Comment:
Class javadoc should be updated to reflect that only one secret is being created.

Reply:
Javadoc has been updated. Thanks!